routers/web/web.go

```diff
@@ -642,6 +642,10 @@ func RegisterRoutes(m *web.Route) {
 			m.Post("/teams/{team}/edit", bindIgnErr(forms.CreateTeamForm{}), org.EditTeamPost)
 			m.Post("/teams/{team}/delete", org.DeleteTeam)
 
+			m.Get("/times/by_repos", org.TimesByRepos)
+			m.Get("/times/by_members", org.TimesByMembers)
+			m.Get("/times/by_milestones", org.TimesByMilestones)
```

Comment:
Please use `m.Group` here.

Reply:
Changed. Please review for correct use of `context.OrgAssignment`!